index.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const SCHEMA_CONFIG = require('screwdriver-data-schema').config.template.template;
+const SCHEMA_PIPELINE_CONFIG = require('screwdriver-data-schema').config.pipelineTemplate.template;
```

Comment:
Consider renaming 
```suggestion
const SCHEMA_PIPELINE_TEMPLATE = require('screwdriver-data-schema').config.pipelineTemplate.template;
```

Also rename, existing `SCHEMA_CONFIG` to `SCHEMA_JOB_TEMPLATE`